### ISSUE TITLE
[Bug report]: Mac Desktop - Failed In-App Update of IOTA Firefly from 1.7.4 to 1.7.5

### ISSUE THREAD

**Issue by Sayvai** [iotaledger/firefly#7422]
### Platform

Desktop

### Version

1.7.5

### Description

I was notified of a new update to Firefly desktop (Mac) from 1.7.4 to 1.7.5, and selected the option from the menu to update to the latest version.

It appears to have internally downloaded the latest update correctly, with a toast to prompt me to restart to complete the latest update installation.

But during the installation process, a system notification notified me that the new installation failed , and OS prompt mentioned that it moved the Firefly desktop app to the bin. See screenshot:

`"Firefly" is damaged and can't be opened. You should move it to the Bin`

![Screenshot 2023-09-13 at 12 00 09](https://github.com/iotaledger/firefly/assets/7581546/c2ee3a6e-c577-4716-a629-6542e92ab8ea)

So i had to re-download the full DMG binary from the Firefly desktop [website](https://firefly.iota.org/) with the latest 1.7.5 version. But even after apparently installing the new binary and then clicking to open Firefly, the system failed to open the application, with the same resulting OS prompt message as seen from the screenshot above.

Then i downloaded the DMG binary from the repo [releases](https://github.com/iotaledger/firefly/releases/tag/desktop-1.7.5) page, and installed, and tried to open the FireFly app, and that also failed, similar to the binary downloaded from the website.

I have a feeling that once another hotfix is released, I will need to actually use my **stronghold backup** for the first time.. hoping that my wallet is restored successfully.. now that the firefly app was unexpectedly removed from my OS.. 😨

### Expected behaviour

1.7.5 to be installed successfully.

### Actual behaviour

1.7.5 update install failed, and deleted the app entirely from my Mac OS.

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

- Open Firefly Desktop 1.7.4
- Notified of new update
- Select option to update
- Internal app download process begins
- Prompted to install and restart application as part of installation
- Received OS prompt that `"Firefly" is damaged and can't be opened. You should move it to the Bin`, with action button options `Move to Bin`, and `Cancel` (I clicked 'Move to Bin')

### Operating System

Mac - Ventura 13.4 (22F66)

### Diagnostics

```shell
Unable to provide diagnostics as OS moved app to bin.
```

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

None

### Crash Status

None of the above

### Error Messages

`"Firefly" is damaged and can't be opened. You should move it to the Bin`

### Error Logs

```shell
N/K
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by begonaalvarezd** [comment]
Thanks for reporting this 🙏🏼 we'll check it out asap

**Comment by DaZRool** [comment]
Same issue on macOS Ventura Version 13.5.2 (22G91)

**Comment by begonaalvarezd** [comment]
It seems like the github action created a corrupted dmg, we are re-triggering the release 🤞🏼

**Comment by Sayvai** [comment]
@begonaalvarezd -  Is there official instructions on how to restore my IOTA firefly wallet data once a hotfix is released to fix the last hotfix?

If so, are you able to provide a link to the instructions?

I have a very recent stronghold backup that was last taken after installing `1.7.4` 2 weeks ago, and so would like to use this stronghold backup file to restore my account / wallet.

**Comment by begonaalvarezd** [comment]
> @begonaalvarezd - Is there official instructions on how to restore my IOTA firefly wallet data once a hotfix is released to fix the last hotfix? I will have to re-install the app and set up my account again..
> 
> If so, are you able to provide a link to the instructions?
> 
> I have a very recent stronghold backup that was last taken after installing `1.7.4` 2 weeks ago, and so would like to use this stronghold backup file to restore my account / wallet.

I dont think you will have to restore your wallet again, you just need to install the new version from github or the website once its released. In any case, you can always restore any profile from stronghold (old or new) or your 24 words mnemonic (or ledger if you use a hardware profile)

**Comment by begonaalvarezd** [comment]
> ~@begonaalvarezd - Is there official instructions on how to restore my IOTA firefly wallet data once a hotfix is released to fix the last hotfix? I will have to re-install the app and set up my account again..~
> 
> ~If so, are you able to provide a link to the instructions?~
> 
> ~I have a very recent stronghold backup that was last taken after installing `1.7.4` 2 weeks ago, and so would like to use this stronghold backup file to restore my account / wallet.~
> 
> I re-downloaded and re-installed the DMG binary for [1.7.4](https://github.com/iotaledger/firefly/releases/tag/desktop-1.7.4), and was able to login with my usual passcode without having to go through a Stronghold restoration process, and so would it seems unlikely that I would need to use my stronghold backup once the latest hotfix rebuild comes out (🤞)

yup, that should be the case, just install and business as usual

**Comment by begonaalvarezd** [comment]
New version is out

### Announcement message

We just released Firefly IOTA 1.7.5 with the hotfix "not display new staking event popup until there is a new one"

:loudspeaker: For macOS users, we had to make the release twice because our github action generated an incorrect artifact. If you already upgraded to v1.7.5 and you received a message like "Firefly is damaged and cant be opened", please download it again from the official website. Your existing profiles will still be there in the new version. 

:arrow_down:  Download:
https://firefly.iota.org/

:bug:  Bug Reporting:
https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug+report&template=bug_report.yml&title=%5BBug+report%5D%3A

**Comment by Sayvai** [comment]
@begonaalvarezd to note, when i tried to update from within the app version 1.7.4, it had failed with a checksum mismatch error message, as seen in screenshot below. You may also want to fix the broken image from within the firefly error message popup model (top-left corner of screenshot):

<img width="912" alt="Screenshot 2023-09-13 at 14 45 10" src="https://github.com/iotaledger/firefly/assets/7581546/e1420f67-e779-4a72-8c2f-617d638fcd3d">
<details>
  <summary>View text snippet of message.</summary>

```
App Version: 1.7.4

platform: macOS
platformVersion: 13.4
platformArchitecture: x64
cpuCount: 8
totalMem: 16384.0 MB
freeMem: 314.8 MB

Error Type: [Main Context] Unhandled Rejection

Error: sha512 checksum mismatch, expected esGNf74su26/pXI4pirhU22a/8JY0vkxBGNqB4LuTAhsNgjwoYoddEImHvMjWSdZIgXZf7J4l4HvLgONgF9pHA==, got SRv8187RhgjR9gn/2DnkioS7J6swoYoGBZfZ40nrqbfSTy5iRa4inzO/RY5xewS2E7tXdFdgDbYZ5m/Xw4xY8g==
```
</details>

I believe the checksum displayed from error message of app version `1.7.4`. Maybe it's because when i re-installed 1.7.4 earlier, it still held a reference of the checksum of the corrupt version of `1.7.5`, before the second build attempt of the DMG release succeeded and was released.

So afterwards, I downloaded app version `1.75` from the FireFly [website](https://firefly.iota.org/), manually verified checksum against the checksum value displayed on the [Releases](https://github.com/iotaledger/firefly/releases/tag/desktop-1.7.5) page which was matched with the downloaded DMG binary, and then successfully installed and opened the app with the account / wallet data persisted.

I suspect those still on the outdated version of `1.7.4` and who typically update in-app, will need to manually download the update from the FireFly [website](https://firefly.iota.org/), or from the [Releases](https://github.com/iotaledger/firefly/releases/tag/desktop-1.7.5) page.
